### DISCUSSION TITLE
Does xlsxreader handle functions properly?

### DISCUSSION THREAD

**Discussion by jsd3** [TheDataShed/xlsxreader#43] [Q&A] [answered]
I'm looking for something exactly like xlsx reader, but some of the cells I need to read data out of have formulas ... can xlsx reader handle formulas?

**Comment by jsd3** [comment, marked as answer]
So far in testing, it appears that the correct values are returned for cells with formulas. If anyone knows of any problems with formulas, please say. Thanks.

**Reply by dglsparsons** [reply]
it should do. The xlsx spec actually stores the calculated values from a formula within the spreadsheet data, with the formula that derived a value being stored alongside it. This means for our purpose we don't actually need to care about formula - we just read the value that is already stored.
